CONTRIBUTING.md

```diff
@@ -0,0 +1,232 @@
+Also somebody may already have started discussing or working on implementing the same or a similar idea,
+so you can cooperate to create a better solution together.
+
+## Open pull requests
+
+Look through the currently open pull requests in the collection repository.
+You can help by reviewing them. Reviews help move pull requests to merge state. Some good pull requests cannot be merged only due to a lack of reviews. For more information how to provide a good review, refer to the [review checklist](REVIEW_CHECKLIST.md).
+And it is always worth saying that good reviews are often more valuable than pull requests themselves.
+
+Also, consider taking up a valuable, reviewed, but abandoned pull request which you could politely ask the original authors to complete yourself.
+
+## Looking for an idea to implement
+
+First, see the paragraphs above.
+
+If you came up with a new feature, it is always worth creating an issue
+before starting to write code to discuss the idea with the community first.
+If you are going to implement the feature yourself, say it in the issue explicitly to avoid working in parallel with somebody else.
+
+## Step-by-step guide how to get into development quickly
+
+We assume that you use Linux as a work environment (you can use a virtual machine as well) and have `git` installed.
+
+1. If possible, make sure that you have installed and started `docker`. While you can also run tests without docker, this makes it a lot easier since you do not have to install the precise requirements, and tests are running properly isolated and in the exact same environments as in CI. You often can also use `podman` with the `docker` executable shim, so if you have that you probably do not need to install `docker`.
+
+2. Clone the [ansible-core](https://github.com/ansible/ansible) repository:
+```bash
+git clone https://github.com/ansible/ansible.git
+```
+
+Instead of installing ansible-core from source, you can also work with an already existing installation of Ansible, ansible-base or ansible-core. Simply skip steps 2 and 3 in that case.
+
+3. Go to the cloned repository and prepare the environment:
+```bash
+cd ansible && source hacking/env-setup
+pip install -r requirements.txt
+cd ~
+```
+4. Create the following directories in your home directory:
+```bash
+mkdir -p ~/ansible_collections/NAMESPACE/COLLECTION_NAME
+```
+
+For example, if the collection is `community.mysql`, it will be:
+```bash
+mkdir -p ~/ansible_collections/community/mysql
+```
+
+If the collection is `ansible.posix`, it will be:
+```bash
+mkdir -p ~/ansible_collections/ansible/posix
+```
+
+5. Fork the collection repository through the GitHub web interface.
+
+6. Clone the forked repository from your profile to the created path:
+```bash
+git clone https://github.com/YOURACC/COLLECTION_REPO.git ~/ansible_collections/NAMESPACE/COLLECTION_NAME
+```
+
+If you prefer to use the SSH protocol:
```

Comment:
I would do it the other way around, start with `ssh`, and hint on `https` as an alternative.

Reply:
newbie contributors often have no idea about it. I'd avoid failings (and resp. confusion) on this step. I'd like to see them happy quickly - they could figure out how ssh works later, IMO